tests/Microsoft.Identity.Test.Unit.net45/CacheTests/CacheSerializationTests.cs

```diff
@@ -427,9 +427,10 @@ public void TestDictionarySerialization()
 
         [TestMethod]
         [DeploymentItem(@"Resources\CacheFromTheFuture.json")]
-        public async Task UnkownNodesTestAsync()
+        public async Task UnknownNodesTestAsync()
         {
             string jsonFilePath = ResourceHelper.GetTestResourceRelativePath("CacheFromTheFuture.json");
+            string jsonFilePath2 = ResourceHelper.GetTestResourceRelativePath("CacheFromTheFuture.json");
```

Comment:
why do you need this second variable with the same path?

Reply:
That's not supposed to be there. I was having issues with the file paths getting picked up and was trying something out but forgot to delete that line :/